package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "to-do_list-app",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack watch --mode=development --config webpack.config.js",
+    "prod": "webpack --mode=production --config webpack.config.js"
+  },
```

Comment:
- [ ] There is no `start` in your script, so I can use `npm start` to fire up your app from the terminal. Kindly look into this in your webpack set up and fix it. Below is my package.json for the same project. You can check and find what you are missing.

```
   {
  "name": "webpack-boilerplate",
  "version": "1.0.0",
  "description": "> I will build a simple but powerful webpack boilerplate, which can later be used as a starting point in my projects.",
  "private": true,
  "scripts": {
    "test": "jest --env=jsdom ",
    "start": "webpack serve --open",
    "build": "webpack"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Qoosim/webpack-boilerplate.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/Qoosim/webpack-boilerplate/issues"
  },
  "homepage": "https://github.com/Qoosim/webpack-boilerplate#readme",
  "devDependencies": {
    "@babel/plugin-transform-modules-commonjs": "^7.16.0",
    "babel-eslint": "^10.1.0",
    "css-loader": "^6.5.0",
    "eslint": "^7.32.0",
    "eslint-config-airbnb-base": "^14.2.1",
    "eslint-plugin-import": "^2.25.2",
    "html-webpack-plugin": "^5.5.0",
    "jest": "^27.3.1",
    "jest-localstorage-mock": "^2.4.18",
    "style-loader": "^3.3.1",
    "stylelint": "^13.13.1",
    "stylelint-config-standard": "^21.0.0",
    "stylelint-csstree-validator": "^1.9.0",
    "stylelint-scss": "^3.21.0",
    "webpack": "^5.60.0",
    "webpack-cli": "^4.9.1",
    "webpack-dev-server": "^4.3.1"
  },
  "dependencies": {
    "lodash": "^4.17.21"
  },
  "jest": {
    "resetMocks": false,
    "setupFiles": [
      "./__setups__/localstorage.js", "jest-localstorage-mock"]
  }
}
```

You can also check my webpack configure to also know what is missing in your webpack config.

```
  const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  mode: 'development',
  entry: './src/index.js',
  devServer: {
    static: './dist',
  },
  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
    clean: true,
  },
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: ['style-loader', 'css-loader'],
      },
    ],
  },
};
```

Reply:
Thank you for the feedback. The recommended changes have been made.